x-pack/plugins/canvas/public/expression_types/arg_types/container_style/border_form.js

```diff
@@ -38,13 +39,29 @@ export const BorderForm = ({ value, radius, onChange, colors }) => {
   return (
     <EuiFlexGroup gutterSize="s">
       <EuiFlexItem grow={2}>
-        <EuiFormRow label="Width" compressed>
+        <EuiFormRow
+          label={
+            <FormattedMessage
+              id="xpack.canvas.expressionTypes.containerStyle.border.FormRowWidthLabel"
```

Comment:
xpack.canvas.expressionTypes.containerStyle.border => xpack.canvas.borderForm for all ids in this file